kubernetes/deployment.yml

```diff
@@ -23,7 +23,13 @@ spec:
             path: config.yml
```

Comment:
Having it doesn't hurt, and it would help anyone who wants to deploy this to Kubernetes